src/internal/bigintToFeeAmount.ts

```diff
@@ -0,0 +1,17 @@
+import { FeeAmount } from "@uniswap/v3-sdk";
+
+const validFeeAmounts = new Set(Object.values(FeeAmount));
+
+export default function bigintToFeeAmount(value: bigint | undefined): FeeAmount | undefined {
```

Comment:
seems weird for this function to take undefined and return undefined. that could be handled outside of this maybe?

Reply:
yeah you're right.